core/src/main/java/org/mskcc/cbio/portal/dao/DaoGeneticEntity.java

```diff
@@ -0,0 +1,55 @@
+package org.mskcc.cbio.portal.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class DaoGeneticEntity {
+
+	/**
+     * Private Constructor to enforce Singleton Pattern.
+     */
+    private DaoGeneticEntity() {
+    }
+    
+    public static enum EntityTypes
```

Comment:
I think this is fine. I would like to keep it as it is for now (also related to my previous comments about this field).